.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Have Release Engineering added as a reviewer to any packaging PR
+* @daos-stack/build-and-release-watchers
```

Comment:
Any idea how to fix this "unknown owner" error?

Reply:
Fixed.